static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -0,0 +1,47 @@
+// @flow
```

Comment:
i think we should get some consensus about the language here. i've been calling the refinement options on the left side of the page 'facets'. here they are (implicitly) called 'filters'. do we have any strong opinions on what these should be called? @pdpinch?